docs/aboutthisbook.adoc

```diff
@@ -14,6 +14,19 @@ The content for this book, including sample code and tests, are sourced from the
 To report a problem (typo, grammar, or technical fault), please https://github.com/heckj/swiftui-notes/issues/new/choose[Open an issue] in GitHub.
 If you are so inclined, feel free to fork the project and send me https://github.com/heckj/swiftui-notes/compare?expand=1[pull requests] with updates or corrections.
 
+== Acknowledgements
+
+Michael Critz designed and provided the cover art.
```

Comment:
@mcritz I can easily add a link reference here if you'd like - to twitter or github, or somewhere else if you'd like.

I wasn't sure if you'd want that, but I absolutely wanted to give you credit for the design in the publication.

Reply:
Thanks for the mention. That’s all that’s necessary for now. We might revisit this later as I build more deliverables.